Palaso/Progress/Progress.cs

```diff
@@ -608,15 +603,7 @@ public bool WasCancelled
 
 		public void Clear()
 		{
-			LastError = LastWarning =LastStatus = string.Empty;
-		}
-	}
-
-	public static class SafeFormatString
-	{
-		public static string EscapeThingsThatLookLikeArguments(this string format, params object[] args)
-		{
-			return format.Replace("{", @"\{");
```

Comment:
not sure this would have worked anyway; MSDN doc says, for String.Format, escape `{` and `}` with `{{` and `}}`.